graphene/types/enum.py

```diff
@@ -63,7 +63,7 @@ def from_enum(
         cls, enum, name=None, description=None, deprecation_reason=None
     ):  # noqa: N805
         name = name or enum.__name__
-        description = description or enum.__doc__
+        description = description or "An enumeration."
```

Comment:
How about:

```python
description = description or enum.__doc__ or "An enumeration."
``` 
instead, to still be able to utilize the docstring descriptions of enums when they _are_ present, like:

```python
>>> class Color(Enum):
...     """my description"""
...     RED = 1
...
>>> Color.__doc__
'my description'
```

Reply:
Awesome!